src/modules/outliers/iqr.js

```diff
@@ -0,0 +1,102 @@
+import { deNormalizerMap } from './normalization'
+
+export const IQR = 'IQR'
+
+export const getQuartilePosition = (data, q) => {
+    const pos = (data.length + 1) / 4
+
+    switch (q) {
+        case 0.75:
+            return pos * 3
+        case 0.5:
+            return pos * 2
+        case 0.25:
+        default:
+            return pos
+    }
+}
+
+export const getQuartileValue = (data, q = 0.25) => {
+    if (data.length < 3) {
```

Comment:
Do you need 3 or 4 items to calculate the quartile? if 4 then this should be `<= 3`, right?

Reply:
Also, what happens if you pass e.g. values like this: `[5, 5, 5, 5]`